modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -59,16 +59,35 @@ Note that Group 2 contains all the replica vBuckets that correspond to active vB
 [#vbucket-distribution-across-unequal-groups]
 === Unequal Groups
 
+A number of constraints come into play when allocating active and replica vbuckets across server groups:
+
+[horizontal]
+
+*rack-zone*:: all replicas of a given vbucket must reside in separate server groups.
+
+*active balance*:: there should be approximately{empty}footnote:approximate-vbuckets[Note that by "approximately the same" we mean the number should vary by 5 vbuckets or less] the same number of active vbuckets on each node in the cluster.
```

Comment:
I think we shouldn't specify a hard number on the max difference of active vbucket counts on different nodes. Generally, the active vbucket counts will only vary by 1 across nodes, but we might loosen that in the future. Also I don't want customers with 100 nodes with an average of 10 vbuckets per node complaining that some nodes might have as many as 15 vbuckets and carry 50% greater load. I'd prefer to just say the the counts are approximately the same and leave it at that.